common/src/main/java/com/wynntils/features/commands/FilterAdminCommandsFeature.java

```diff
@@ -43,10 +44,14 @@ public void onCommandPacket(CommandsAddedEvent event) {
 
         RootCommandNode<SharedSuggestionProvider> newRoot = new RootCommandNode<>();
         for (CommandNode<SharedSuggestionProvider> child : root.getChildren()) {
-            if (!FILTERED_COMMANDS.contains(child.getName())) {
-                newRoot.addChild(child);
+            // Only add literal nodes, not argument nodes
```

Comment:
This is the direct children to the root, yes? What are the argument nodes here? I thought it was only possible to have literals as the top-most children.

Reply:
You can also have argument nodes, which are kind of weird, they don't resolve when trying to calculate autocomplete, freezing the whole process. It is added by creating a new root command node, I am not really sure why. Either way, this should work..